helmfile/values/velero.yaml.gotmpl

```diff
@@ -1,7 +1,6 @@
-installCRDs: false
```

Comment:
This value for example is not present in the version that we are using https://github.com/vmware-tanzu/helm-charts/blob/velero-2.8.2/charts/velero/values.yaml